upgrades/v1.2.0.md

```diff
@@ -0,0 +1,45 @@
+# Kroma `v1.2.0` Upgrade Instructions
+
+This document provides instructions to upgrade from Kroma `v1.1.1` to `v1.2.0`.
+For more details about `v1.2.0`, please refer the [release note](https://github.com/kroma-network/kroma/releases/tag/v1.2.0).
+This upgrade includes upstream of Optimism [`v1.3.0`](https://github.com/ethereum-optimism/optimism/releases/tag/op-node%2Fv1.3.0).
+This upgrade includes Canyon hardfork, so all full nodes and validators must complete the upgrade before the Canyon 
+upgrade is activated.
+
+The scheduled activation time of Canyon on Kroma sepolia is `Wed Feb 14 2024 08:00:00 GMT+0000`.
+
+## Guides for upgrade
+
+### Stop Kroma
+
+For upgrade, stop your Kroma full node or validator.
+```bash
+# for full node
+docker compose --profile fullnode down
+
+# for validator
+docker compose --profile validator down
+```
+
+### Common
+
+You need to update your `.env` file.
+
+First, update the tag of `kroma-node` and `kroma-validator` docker image.
+```
+IMAGE_TAG__KROMA_GETH=v0.4.2
+IMAGE_TAG__KROMA_NODE=v1.2.0
+IMAGE_TAG__KROMA_VALIDATOR=v1.2.0
+```
```

Comment:
`rollup.json` should be changed also!

Reply:
Thanks! it is added here [19ccbe0](https://github.com/kroma-network/kroma-up/pull/23/commits/19ccbe048a8f33e9cbdccfbc1d65b1602e382ce6)